generators/app/templates/src/app/core/authentication/__auth._authentication.guard.spec.ts

```diff
@@ -9,13 +14,27 @@ describe('AuthenticationGuard', () => {
   let authenticationGuard: AuthenticationGuard;
   let credentialsService: MockCredentialsService;
   let mockRouter: any;
+<% if (props.tools.includes('jest')) { -%>
+  let mockSnapshot: any;
```

Comment:
😭 hopefully it will be fixed at some point, for ref: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/32901